core/src/main/java/org/ehcache/core/internal/service/ServiceLocator.java

```diff
@@ -438,11 +457,22 @@ ServiceMap lookupDependenciesOf(ServiceMap resolved, Class<? extends Service> re
   private static Set<Class<? extends Service>> identifyTransitiveDependenciesOf(final Class<?> clazz) {
 
     Set<Class<? extends Service>> dependencies = identifyImmediateDependenciesOf(clazz);
+    for (Class<? extends Service> dep : dependencies) {
```

Comment:
Again, I don't like the variables naming. Those are classes, not dependencies

Reply:
I have kept teh existing naming that what was already there elsewhere in the same class.